packages/cli/src/lib/collect/collect-command.ts

```diff
@@ -18,6 +18,19 @@ export function yargsCollectCommandObject(): CommandModule {
       console.info(chalk.bold(CLI_NAME));
       console.info(chalk.gray(`Run ${command}...`));
       await collectAndPersistReports(options);
+      const { upload = {} } = args as unknown as Record<
+        'upload',
+        object | undefined
+      >;
+      if (Object.keys(upload).length === 0) {
+        console.info(`
+        💡 Visualize your reports:
+        - npx code-pushup upload - Run upload to upload the created report to the server
+          https://github.com/code-pushup/cli/tree/main/packages/cli#upload-command
+        - npx code-pushup autorun - Run collect & upload
+          https://github.com/code-pushup/cli/tree/main/packages/cli#autorun-command
+        `);
```

Comment:
This will produce several levels of identation in the terminal output. (Same issue as mentioned in https://github.com/code-pushup/cli/pull/437#discussion_r1469374238.)

Possible solutions (last one is my favourite):

```suggestion
        console.info(`💡 Visualize your reports:
- npx code-pushup upload - Run upload to upload the created report to the server
  https://github.com/code-pushup/cli/tree/main/packages/cli#upload-command
- npx code-pushup autorun - Run collect & upload
  https://github.com/code-pushup/cli/tree/main/packages/cli#autorun-command`);
```

or

```suggestion
        console.info(
          '💡 Visualize your reports:\n' +
          '- npx code-pushup upload - Run upload to upload the created report to the server\n' +
          '  https://github.com/code-pushup/cli/tree/main/packages/cli#upload-command\n' +
          '- npx code-pushup autorun - Run collect & upload\n' +
          '  https://github.com/code-pushup/cli/tree/main/packages/cli#autorun-command'
        );
```

or

```suggestion
        console.info([
          '💡 Visualize your reports:',
          '- npx code-pushup upload - Run upload to upload the created report to the server',
          '  https://github.com/code-pushup/cli/tree/main/packages/cli#upload-command',
          '- npx code-pushup autorun - Run collect & upload',
          '  https://github.com/code-pushup/cli/tree/main/packages/cli#autorun-command',
        ].join('\n'));
```